previousnext/docker-builder-mysql
# #48652 - Add command to add SQL dumps to builder queue.

To do

- [x] Improve user feedback / output / error messages

# Testing

1. Check out this branch
1. Run `make build`
1. Ensure AWS credentials loaded into scope.
1. Run command below, with parameters adjusted for your use case.

```sh
./bin/docker-builder-mysql_darwin_amd64 queue \
  --sql /tmp/db.sql 
  --bucket=docker-builder-mysql-pnx \
  --image-repo=previousnext/pnx-d8-db \
  --region=us-east-1
```

And you should get this kind of output.

```sh
Uploading file /tmp/db.sql  to previousnext/pnx-d8-db/latest--2018-01-10T17-23.sql
Successfully added the SQL dump to the builder queue -  https://docker-builder-mysql-pnx.s3.amazonaws.com/previousnext%2Fpnx-d8-db%2Flatest--2018-01-10T17-23.sql
```
